uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/CamelEditor.java

```diff
@@ -175,7 +175,7 @@ public void addCamelComponent(CamelComponent component, String route) {
 		log.debug("Adding '" + component + "' component into '" + route + "' route");
 		Point r = getInEditorCoords(route);
 		addCamelComponent(component, r.x + 10, r.y + 10);
-		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		new WaitWhile(new JobIsRunning(), TimePeriod.VERY_LONG);
```

Comment:
it was already 1 minute long, seems quite a lot.
To see if it might reveal a performance issue in product itself:
does the time to add a camel component increased recently?
have you noticed if it is for specific Camel component?

Reply:
in the past adding components to the diagram which had no dependency entry in the pom yet caused a little more delay. Not sure how it is behaving atm.

Reply:
OK, it make sense. I will raise an issue because smoke tests sometimes fails with the following error:
```Timeout after: 60 s.: The following jobs are still running: loading http://camel.apache.org/schema/spring/camel-spring.xsd```
(_Otherwise_ component was added into the route)

Reply:
https://issues.jboss.org/browse/FUSETOOLS-2793

Reply:
the xsd loading happens usually later in process when validating I think